src/main/java/racing/view/ResultView.java

```diff
@@ -1,25 +1,47 @@
 package racing.view;
 
-import java.util.stream.IntStream;
+import java.text.MessageFormat;
+import java.util.List;
 
+import racing.domain.Car;
 import racing.domain.RacingCars;
 
 public class ResultView {
 
+	private static final String COMMA_DELIMETER = ",";
+	private static final String WINNER_PRINT_FORMAT = "{0}가 최종 우승하였습니다.";
+	private static final String NO_WINNER_PRINT_FORMAT = "우승자가 없습니다.";
+
 	public ResultView() {
 	}
 
-	public void printCarLocations(RacingCars racingCars) {
-		IntStream.range(0, racingCars.getRacingCars().size())
-			.forEach(i -> {
-				printCarLocation(racingCars.getRacingCars().get(i).getLocation());
-			});
+	public void printCarStatuses(RacingCars racingCars) {
+		for (int i = 0; i < racingCars.getRacingCars().size(); i++) {
+			printCarName(racingCars.getRacingCars().get(i));
+			printCarLocation(racingCars.getRacingCars().get(i));
+		}
+	}
+
+	private void printCarName(Car car) {
+		System.out.print(car.getName() + " : ");
 	}
 
-	private void printCarLocation(int location) {
-		IntStream.range(0, location)
-			.mapToObj(i -> "-")
-			.forEach(System.out::print);
+	private void printCarLocation(Car car) {
+		for (int i = 0; i < car.getLocation(); i++) {
+			System.out.print("-");
+		}
 		System.out.println();
 	}
+
+	public void printWinners(List<Car> winners) {
+		if (winners.isEmpty()) {
+			System.out.println(NO_WINNER_PRINT_FORMAT);
+			return;
+		}
+		CharSequence[] winnerNames = winners.stream()
+			.map(Car::getName)
+			.toArray(CharSequence[]::new);
+
+		System.out.print(MessageFormat.format(WINNER_PRINT_FORMAT, String.join(COMMA_DELIMETER, winnerNames)));
```

Comment:
```suggestion
        String winnerNames = winners.stream()
                .map(Car::getName)
                .collect(Collectors.joining(COMMA_DELIMETER));

        System.out.print(MessageFormat.format(WINNER_PRINT_FORMAT, winnerNames));
```
java8 이후로부터는 코딩하면서 array를 직접 만질 일이 많지 않은 것 같아요. 혹시라도 array가 보인다면 다른 방법은 없나 한 번 찾아보셔도 좋을 것 같습니다!